insights/parsers/systemctl_show.py

```diff
@@ -244,6 +247,90 @@ class SystemctlShowCinderVolume(SystemctlShow):
     pass
 
 
+@parser(Specs.systemctl_show_all_services_with_cpuaccounting)
+class SystemctlShowAllServiceWithCPUAccounting(SystemctlShowServiceAll):
+    """
+    Class for parsing the output of command ``systemctl show *.service --all --property=Names,CPUAccounting,CPUQuotaPerSecUSec,CPUShares,StartupCPUShares,UnitFileState,SubState``.
+
+    Sample Input::
+
+        CPUAccounting=no
+        CPUShares=18446744073709551615
+        StartupCPUShares=18446744073709551615
+        CPUQuotaPerSecUSec=infinity
+        Names=test1.service
+        SubState=dead
+        UnitFileState=static
+
+        CPUAccounting=yes
+        CPUShares=18446744073709551615
+        StartupCPUShares=18446744073709551615
+        CPUQuotaPerSecUSec=infinity
+        Names=test2.service
+        SubState=dead
+        UnitFileState=enabled
+
+    Sample Output::
+
+        {'test1.service': {
+            'Names': 'test1.service',
+            'SubState': 'dead',
+            'UnitFileState': 'static',
+            'CPUQuotaPerSecUSec': 'infinity',
+            'CPUShares': '18446744073709551615',
+            'StartupCPUShares': '18446744073709551615',
+            'CPUAccounting': 'no'
+            },
+        'test2.service': {
+            'Names': 'test2.service',
+            'SubState': 'dead',
+            'UnitFileState': 'enabled',
+            'CPUQuotaPerSecUSec': 'infinity',
+            'CPUShares': '18446744073709551615',
+            'StartupCPUShares': '18446744073709551615',
+            'CPUAccounting': 'yes'
+            }
+        }
+
+    Attributes:
+        services_with_cpuaccouting_enabled (list): A list of service names which enabled CPUAccouting.
+
+    Raises:
+        SkipComponent: If there is no service enabled CPUAccouting.
+
+    Examples:
+        >>> 'test1.service' in all_services_with_cpuaccouting_info
+        True
+        >>> all_services_with_cpuaccouting_info.get('test1.service').get('CPUAccounting', None)
+        'no'
+        >>> 'test2.service' in all_services_with_cpuaccouting_info.services_with_cpuaccouting_enabled
+        True
+    """
+
+    def parse_content(self, content):
+        super(SystemctlShowAllServiceWithCPUAccounting, self).parse_content(content)
+        self.services_with_cpuaccouting_enabled = []
+        for service_name, service_info in self.items():
```

Comment:
By doing this, it makes the parser a bit specific and limited for cpuaccounting only, I think it would be better to keep the `__init__` part of this parser untouched, but add one/two `get` methods to get what you want to provide specifically for `CPUAccounting`, how do you think?

Reply:
@xiangce I updated it, please help to review.